modules/Project/summary_blocks/Milestones.php

```diff
@@ -31,6 +33,16 @@ public function process(Vtiger_Record_Model $recordModel)
 		$total = $query->count();
 		$open = $query->andWhere(['vtiger_projectmilestone.projectmilestone_status' => ['PLL_IN_PROGRESSING', 'PLL_IN_APPROVAL']])->count();
 		\App\Log::trace('Exiting Tasks::process() method ...');
-		return ['open' => $open, 'total' => $total];
+		return [
+			[
+				'lable' => \App\Language::translate('LBL_OPEN'),
```

Comment:
```suggestion
				'label' => \App\Language::translate('LBL_OPEN'),
```
typo in several places